lib/plugins/docker/index.js

```diff
@@ -4,13 +4,25 @@ import preset from '../../pattern-preset';
 
 export default class Docker {
 
+  static clickHandler({ image }) {
+    let isOffical = true;
+    const imageName = image.split(':')[0];
+
+    if (image.includes('/')) {
+      isOffical = false;
+    }
+
+    return [
+      `https://hub.docker.com/${isOffical ? '_' : 'r'}/${imageName}`,
+    ];
+  }
+
   getPattern() {
     return preset('Docker');
   }
 
   parseBlob(blob) {
-    insertLink(blob.el, DOCKER_FROM, {
-      resolver: 'dockerImage',
+    insertLink(this, blob.el, DOCKER_FROM, {
```

Comment:
Yeah, I was thinking about the duplication as well, when I was writing the comment. It's not ideal, but I think it's worth the gains in clarity that putting it in the map gives.

If you'd like, you could instead pass `plugin: this` to `insertLink`, and `insertLink` could delete the `plugin` key and add a `pluginName` key, to avoid duplicating `.constructor.name`, but I'm not sure if that's better...

I definitely agree on loving code reviews 👍